src/components/order-service/order-service.controller.ts

```diff
@@ -154,4 +180,8 @@ export class OrderServiceController implements angular.IController {
     let step = _.find(this.ctrl.steps, { id: id });
     this.gotoStep(step);
   }
+
+  private isNewProject(): boolean {
+    return this.ctrl.selectedProject && this.ctrl.selectedProject.metadata ? this.ctrl.selectedProject.metadata.uid === 'create_project' : false;
```

Comment:
I'd rather leave `uid` undefined instead of setting it to something else. Then a project without a `uid` is new.

I know this will never actually happen, but technically `create_project` is a valid UID for a real project :)